### DISCUSSION TITLE
Better text hightlighthing in editor

### DISCUSSION THREAD

**Discussion by lubomirblazekcz** [outline/outline#2866] [Ideas]
Current text hightlighthing needs some improvements, text is very monochrone overall.

1. `<code>` blocks are really hard to distinguish from regular text, you should consider adding some background at least to make it more readable (look at how it looks in github dark/light mode, much more readable)
2. consider adding accent colors for links, either the default blue or the accent color from collection (or make that adjustable )

I think these two changes would greatly improve readability

**Comment by lubomirblazekcz** [comment]
Here are example images with `<code>` blocks

**Outline code hightlighthing**
![obrazek](https://user-images.githubusercontent.com/6872956/146542387-f8b054b2-7b7b-4e5d-898d-373ab7e82c85.png)

**Github code hightlighthing**
![obrazek](https://user-images.githubusercontent.com/6872956/146542411-61fe79f0-7d48-438b-b5e8-3fcbf1511736.png)

**Reply by tommoor** [reply]
Similar issue filed here this week: https://github.com/outline/outline/issues/2835
